api/source/service/AssetService.js

```diff
@@ -1021,7 +1021,7 @@ exports.cklbFromAssetStigs = async function cklbFromAssetStigs (assetId, stigs)
         revisionStrResolved = `V${resultGetBenchmarkId[0].version}R${resultGetBenchmarkId[0].release}`
       }
       else {
-        let revParse = /V(\d+)R(\d+(\.\d+)?)/.exec(revisionStr)
+        let revParse = dbUtils.extractRevisionDetails(revisionStr)
         revId =  `${benchmarkId}-${revParse[1]}-${revParse[2]}`
```

Comment:
There are several instances where this structure is used too, and these will need to change to the 0,1 items in the array as well.